package.json

```diff
@@ -7,6 +7,7 @@
   },
   "dependencies": {
     "@aeternity/aepp-sdk": "^12.1.2",
+    "@aeternity/aeproject": "^4.1.5",
```

Comment:
isn't this cycle dep?

Reply:
I also thought about that. but probably NPM doesn't care 😆

Reply:
no, aeproject doesn't depend on cli

Reply:
I think dincho and I are talking about the SDK dependency. at least I was thinking about that one

Reply:
the sdk doesn't depend on cli either

Reply:
had a wrong thinking on that. nevermind 😛

Reply:
For sure we already have one: calldata -> cli -> sdk -> calldata

I don't see problems with cycle dependencies, I assume as far as all packages compatible with non-breaking updates (uses `^` in version number), then npm would be able to install a single version of each dependency.

Later I would copy `utils.getFilesystem` to SDK

Reply:
calldata does not depends on the cli perse, it uses it in some helper scripts.

My comment was about aeproject, I was assuming it uses the CLI. Anyway dependancy on aeproject still sounds weird to me :)